providers/deserializers.inc

```diff
@@ -13,6 +13,7 @@
 
 #ifndef OPENSSL_NO_DH
     DESER("DH", "yes", "der", der_to_dh_deserializer_functions),
+    DESER("X9.42 DH", "yes", "der", der_to_dhx_deserializer_functions),
```

Comment:
"X9.42 DH" rather than "DHX"?  How...  quaint

Reply:
Changed - That is the legacy name that we cant get rid of.. Doesnt seem to hurt changing it to DHX.